src/pydata_sphinx_theme/assets/styles/variables/_icons.scss

```diff
@@ -21,4 +21,6 @@ html {
   --pst-icon-github: "\f09b"; // fa-brands fa-github
   --pst-icon-gitlab: "\f296"; // fa-brands fa-gitlab
   --pst-icon-share: "\f064"; // fa-solid fa-share
+  --pst-icon-text: "\f039"; // fa-solid fa-text-justify
```

Comment:
To me this looks too much like a hamburger menu icon. Here are a few ideas for other "generic admonition" icons:

- https://fontawesome.com/icons/star?s=solid&f=classic
- https://fontawesome.com/icons/note-sticky?f=classic&s=solid
- https://fontawesome.com/icons/message?f=classic&s=solid

Reply:
I'm in for the bell